config/rbac/role.yaml

```diff
@@ -13,14 +13,15 @@
 # limitations under the License.
 ---
 apiVersion: rbac.authorization.k8s.io/v1
-kind: Role
+kind: ClusterRole
 metadata:
   name: controller-role
 rules:
   - apiGroups:
       - ""
     resources:
       - configmaps
+      - namespaces
```

Comment:
I think you may have forgotten the kubebuilder annotation (`+kubebuilder:rbac`) for the `namespaces` resource. Perhaps add it as part of the servingruntime_controller.go rbac annotations?

Reply:
Yes, updated role.yaml and put annotations in service_controller.go.